docs/admin/federation-apiserver.md

```diff
@@ -18,11 +18,6 @@
 If you are using a released version of Kubernetes, you should
 refer to the docs that go with that version.
 
-<!-- TAG RELEASE_LINK, added by the munger automatically -->
-<strong>
-The latest release of this document can be found
-[here](http://releases.k8s.io/release-1.3/docs/admin/federation-apiserver.md).
-
```

Comment:
@rhcarvalho Hm I will take a look as soon as I can. I am currently attempting to resolve git conflicts on this locally, however I have not been able to build kubernetes (Fedora 23) recently due to  these issues happening once again: https://bugzilla.redhat.com/show_bug.cgi?id=1038683 https://github.com/coreos/etcd-ca/issues/22

As soon as I am able to find a work-around for these issues and build again, I think the easiest way to ensure a proper "rebase" of this branch with master will be to re-implement my changes on a clean branch, then re-create this branch to keep updating this PR